scapy/contrib/gtp_v2.py

```diff
@@ -603,7 +645,7 @@ class IE_IMSI(gtp.IE_Base):
 class IE_Cause(gtp.IE_Base):
     name = "IE Cause"
     fields_desc = [ByteEnumField("ietype", 2, IEType),
-                   ShortField("length", 6),
+                   ShortField("length", None),
```

Comment:
Why are you changing this value?

Reply:
@guedou Because As per the specs, this is not fixed length IE. So, we cannot hardcode 6 in the length section.

FYI, this change got introduced recently.
Please take a look at here - https://github.com/secdev/scapy/pull/2491/files
I also commented on it saying this is not a correct value.

Reply:
In that case, is there a reason why you do not use a `FieldLenField`?

Reply:
Okay, let me try that.!

Reply:
Hi @p-l- , I've made the changes as requested. Can you please re-review these, Thanks.!